appveyor.yml

```diff
@@ -22,7 +22,7 @@ install:
   - "python -c \"import struct; print(struct.calcsize('P') * 8)\""
 
   # Install the dependencies of the project (skip nibabel for speed)
-  - "conda install --yes --quiet setuptools numpy scipy matplotlib scikit-learn nose mayavi pandas h5py PIL patsy"
+  - "conda install --yes --quiet setuptools numpy scipy matplotlib scikit-learn nose mayavi pandas h5py PIL patsy pyside"
```

Comment:
Pyside is one that works and stops appveyor from raising an error. I think other QT bindings should work too. WX has not been supported by ETS for a while; it will open the GUI but not look right. I don't know about other backends.